src/gui/editlocallyjob.cpp

```diff
@@ -29,12 +29,12 @@ namespace OCC {
 Q_LOGGING_CATEGORY(lcEditLocallyJob, "nextcloud.gui.editlocallyjob", QtInfoMsg)
 
 EditLocallyJob::EditLocallyJob(const QString &userId,
-                                       const QString &relPath,
+                                       const QString &fullRelPath,
```

Comment:
I should have clarified -- it's not clear what a "full relative path" is without digging through how this comes to be created. IMO it should have a more descriptive name or a comment in the header explaining what this parameter is supposed to be, as without context, a "full relative path" can seem to be a bit paradoxical